dbclient/SQLAnalyticsClient.py

```diff
@@ -0,0 +1,103 @@
+import time
+from dbclient import *
+from datetime import datetime as dt
+from datetime import datetime, date
```

Comment:
Minor nit, you've imported datetime as dt, but import it normally below which might cause some confusion when you use which.